src/GPRS.h

```diff
@@ -89,6 +112,7 @@ class GPRS : public ModemUrcHandler {
   int ping(IPAddress ip, uint8_t ttl = 128);
 
   GSM3_NetworkStatus_t status();
+  int getReadyState();
```

Comment:
I'm not too keen on exposing the ready state internals. Is this needed?

Reply:
Not, is not really needed... I used to know the progress of the connection while debugging. Also, I've noticed using GSM::ready() returns 1 only when the progress has finished... so It should be removed.